src/context/ThemeContext.tsx

```diff
@@ -0,0 +1,36 @@
+import { createContext, useState, useContext, useEffect } from 'react';
+
+type Theme = {
+	theme: 'light' | 'dark';
+	setTheme: React.Dispatch<React.SetStateAction<'light' | 'dark'>>;
+};
+
+const ThemeContext = createContext<Theme>({
+	theme: 'light',
+	setTheme: () => null,
+});
+
+// const ThemeContext = createContext({});
+
+export const ThemeProvider: React.FC = ({ children }) => {
+	const [theme, setTheme] = useState<'light' | 'dark'>('light');
+	useEffect(() => {
+		const root = window.document.documentElement;
```

Comment:
I would suggest finding some other way, accessing DOM directly is not advisable in React

Reply:
Tailwind accesses the className in the html tag, There is no other way to toggle the class in the html tag